packages/babel-plugin-proposal-class-static-block/src/index.js

```diff
@@ -0,0 +1,30 @@
+import { declare } from "@babel/helper-plugin-utils";
+import syntaxClassStaticBlock from "@babel/plugin-syntax-class-static-block";
+
+export default declare(({ types: t, template, assertVersion }) => {
+  // todo remove this check after Babel 7.12.0 is published
+  if (process.env.NODE_ENV !== "test") {
+    assertVersion("^7.12.0");
```

Comment:
As a new package it requires `@babel/core` 7.12.0. It depends on `@babel/type` 7.12.0 and `@babel/parser` 7.12.0.